app/models/category_localization.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CategoryLocalization < ActiveRecord::Base
+  belongs_to :category
+
+  validates :locale, presence: true, length: { maximum: 20 }
+  validates :name, presence: true, length: { maximum: 50 }
```

Comment:
While the maximum length of `Category#name` is 50, I do think that certain locales might require a longer length.